_includes/analytics-features.html

```diff
@@ -0,0 +1,98 @@
+                con nuestros datos apoyamos el crecimiento de tu negocio
+            </div>
+        </div>
+    </div>
+
+    <div class="container">
+        <div class="row ">
+            <div class="col text-center p-1 p-md-5">
+                <h2>Toda la potencia de las SOLUCIONES <span class="analytics-brand">Frogtek Analytics</span>
+                    a tu alcance</h2>
+            </div>
+        </div>
+
+        <div class="row analytics-features">
+
+            <div class="col-12 col-md-4">
+                <div class="row d-flex flex-wrap align-items-top">
+                    <div class="col-2"><i class="icon icon-check d-inline-block"></i></div>
+                    <div class="col"><h3>Solución completa<br/>(de extremo a extremo)</h3></div>
+                </div>
+                <p>
+                    Controlamos el proceso desde el entrenamiento del tendero hasta la visualización de los
+                    reportes.
+                </p>
+            </div>
+
+            <div class="col-12 col-md-4">
+                <div class="row d-flex flex-wrap align-items-top">
+                    <div class="col-2"><i class="icon icon-check d-inline-block"></i></div>
+                    <div class="col"><h3>Red propia de puntos de venta</h3></div>
+                </div>
+                <p>
+                    Más de 1,500 tiendas en 12 ciudades de la República Mexicana y creciendo.
+                </p>
+            </div>
+
+            <div class="col-12 col-md-4">
+                <div class="row d-flex flex-wrap align-items-top">
+                    <div class="col-2"><i class="icon icon-check d-inline-block"></i></div>
+                    <div class="col"><h3>Datos de calidad</h3></div>
+                </div>
+                <p>
+                    Garantizamos que las tiendas que usamos en nuestro panel pasan por el
+                    escáner el 95% de sus ventas.
+                </p>
+            </div>
+
+            <div class="col-12 col-md-4">
+                <div class="row d-flex flex-wrap align-items-top">
+                    <div class="col-2"><i class="icon icon-check d-inline-block"></i></div>
+                    <div class="col"><h3>Con todo detalle</h3></div>
+                </div>
+                <p>
+                    Flexibilidad en la agrupación que necesites, manejamos el mínimo nivel de detalle:
+                    producto (sku), transacción, hora, ciudad...
+                </p>
+            </div>
+
+            <div class="col-12 col-md-4">
+                <div class="row d-flex flex-wrap align-items-top">
+                    <div class="col-2"><i class="icon icon-check d-inline-block"></i></div>
+                    <div class="col"><h3>Rápidos</h3></div>
+                </div>
+                <p>
+                    Obtenemos los datos de los tenderos en tiempo real, nuestros insights están disponibles en
+                    una
```

Comment:
Maybe you can omit all these unnecessary line breaks here too

Reply:
👍

Reply:
👍